test/meson.build

```diff
@@ -181,6 +62,20 @@ test_data_files = '''
         unstoppable.service
 '''.split()
 
+if git.found()
+        out = run_command(
+                git,
+                '--git-dir=@0@/.git'.format(meson.source_root()),
+                 'ls-files', ':/test/test-execute/*.service')
+else
+        out = run_command(
+                'sh', '-c', 'ls @0@/test-execute/*.service'.format(meson.current_source_dir()))
```

Comment:
If I’m not mistaken, this could also use `echo` instead of `ls`, right? (Or `printf '%s\n'` to guard against special characters, I suppose.) Because the file list is generated by the glob, not by `ls`, as far as I can tell.